frontend/packages/dev-console/src/components/import/import-validation-utils.ts

```diff
@@ -64,6 +64,9 @@ export const detectGitType = (url: string): GitProvider => {
   if (hasDomain(url, 'gitlab.com')) {
     return GitProvider.GITLAB;
   }
+  if (hasDomain(url, 'gitea.com') || url.includes('gitea')) {
+    return GitProvider.GITEA;
```

Comment:
this doesn't select all gitea instances, e.g., codeberg.org. another way that could be more reliable is to use the gitea API, e.g., https://codeberg.org/api/v1/version will return the gitea version. 

but then there are security implications with leaking user ip addresses if this is done in the frontend so :shrug: 

also i don't think gitea.com hosts any gitea instances (probably like how joinmastodon.org doesn't host a mastodon instance), their own codebase is actually [on github](https://github.com/go-gitea/gitea) lol

Reply:
Through this, the goal is to auto-detect the Git Type only for the URLs that have the "gitea" keyword.
Normally, having the application name in the URLs is a good practice. We are trying to leverage that to detect the git type.